src/view-utilities.js

```diff
@@ -160,67 +160,66 @@ var viewUtilities = function (cy, options) {
       if( shiftKeyDown == true){
         rect_start_pos_x = event.position.x;
         rect_start_pos_y = event.position.y;
-
       }
     });
 
     cy.one('tapend', function(event){
       rect_end_pos_x = event.position.x;
       rect_end_pos_y = event.position.y;
-      //Find top left of rectangle
-      if( rect_start_pos_x > rect_start_pos_x){
+      //check whether corners of rectangle is undefined
+      //abort marquee zoom if one corner is undefined
+      if( rect_start_pos_x == undefined || rect_end_pos_x == undefined){
+        cy.autounselectify(false);
+        cy.elements().unselect();
+        return; 
+      }
+      //Reoder rectangle postions
+      //Top left of the rectangle (rect_start_pos_x, rect_start_pos_y)
+      //right bottom of the rectangle (rect_end_pos_x, rect_end_pos_y)
+      if(rect_start_pos_x > rect_end_pos_x){
         var temp = rect_start_pos_x;
         rect_start_pos_x = rect_end_pos_x;
         rect_end_pos_x = temp;
       }
-      if( rect_start_pos_y > rect_end_pos_y){
+      if(rect_start_pos_y > rect_end_pos_y){
         var temp = rect_start_pos_y;
         rect_start_pos_y = rect_end_pos_y;
         rect_end_pos_y = temp;
       }
-      //Calculate zoom level
-      var zoomLevel = Math.min( cy.width()/ ( Math.abs(rect_end_pos_x- rect_start_pos_x)), 
-        cy.height() / Math.abs( rect_end_pos_y - rect_start_pos_y));
+
+      //Extend sides of selected rectangle to 100px if less than 100px
+      if(rect_end_pos_x - rect_start_pos_x < 100){
+        var extendPx = ( 100 - (rect_end_pos_x - rect_start_pos_y)) / 2;
```

Comment:
Must be `var extendPx = ( 100 - (rect_end_pos_x - rect_start_pos_x)) / 2;`

Reply:
@hasanbalci fixed.